cmake/OpenCVCompilerOptimizations.cmake

```diff
@@ -390,9 +391,12 @@ elseif(RISCV)
   set(CPU_BASELINE "DETECT" CACHE STRING "${HELP_CPU_BASELINE}")
 
 elseif(LOONGARCH64)
+  ocv_update(CPU_LSX_TEST_FILE "${OpenCV_SOURCE_DIR}/cmake/checks/cpu_lsx.cpp")
   ocv_update(CPU_LASX_TEST_FILE "${OpenCV_SOURCE_DIR}/cmake/checks/cpu_lasx.cpp")
-  ocv_update(CPU_KNOWN_OPTIMIZATIONS "LASX")
+  ocv_update(CPU_KNOWN_OPTIMIZATIONS "LSX;LASX")
+  ocv_update(CPU_LSX_FLAGS_ON "-mlsx")
   ocv_update(CPU_LASX_FLAGS_ON "-mlasx")
+  set(CPU_BASELINE "LSX" CACHE STRING "${HELP_CPU_BASELINE}")
   set(CPU_BASELINE "LASX" CACHE STRING "${HELP_CPU_BASELINE}")
```

Comment:
It doesn't make sense to set the same variable twice.

Reply:
Thanks for your review. This is indeed a problem that will cause only lsx to be turned on and lasx to not be turned on. I will find a suitable way to handle it, and then modify it.

Reply:
I changed the way to enable LASX by default by determining the CPU_BASELINE_DISABLE value.